test/Microsoft.AspNet.Mvc.TagHelpers.Test/LinkTagHelperTest.cs

```diff
@@ -298,12 +287,11 @@ public static TheoryData RunsWhenRequiredAttributesArePresent_Data
             helper.Process(context, output);
 
             // Assert
-            Assert.Null(output.TagName);
```

Comment:
Suspect "Runs" in the names of these tag helpers isn't helping us here. Instead think of this set of tests as verifying the tag helper behaviour in positive scenarios. So we want to check what the tag helper did, not just that it did something.